src/test/java/org/fundacionjala/coding/wilson/PlanetTwisted37Test.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.wilson;
+import java.util.Arrays;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author wiltrouble
+ */
+
+public class PlanetTwisted37Test {
+
+    private final Integer[] numbersTest1Exp = new Integer[]{1, 2, 7, 4, 5, 6, 3, 8, 9};
+    private final Integer[] numbersTest2Exp = new Integer[]{12, 14, 13};
+    private final Integer[] numbersTest3Exp = new Integer[]{2, 7, 4, 3, 9};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)